### ISSUE TITLE
About training ViT-Lite

### ISSUE THREAD

**Issue by maodaming** [SHI-Labs/Compact-Transformers#88]
Hello, thank you for your nice work. I found the checkpoint files of ViT-Lite here and successfully achieved the desired accuracy through inference. I wonder if there is any corresponding training code for obtaining the ViT checkpoints? It seems that the example only provides training code for CCT. Thanks a lot!
<img width="473" alt="微信图片_20241213103110" src="https://github.com/user-attachments/assets/ae527192-e22c-475e-ab16-2c812da26600" />
![image](https://github.com/user-attachments/assets/aafde919-2bed-4318-84e6-7fcb2942987e)

**Comment by stevenwalton** [comment]
Training script is the same. You just need to change `--model` and whichever other parameters you wish. You can see the names of the models and their defaults [here](https://github.com/SHI-Labs/Compact-Transformers/blob/main/src/vit.py#L89-L194)

**Comment by maodaming** [comment]
OK, I got it. Thanks for your answering!
